src/plugin/components.js

```diff
@@ -1,27 +1,34 @@
 /**
  * Created by Vouill on 09/07/17.
  */
-import { vueBulmaDefaultRenderElement, camelCaseToDash } from './helpers'
+import { camelCaseToDash, isBulmaAttribute, getOutrEl } from './helpers'
 
 export const componentGenerator = (name, reqOuterElement) => ({
   name,
```

Comment:
this name :)

Reply:
I'm so sorry

Reply:
And I did not want commit the changes of readme file - I'm a bit scattered

Reply:
Its fine no worries ! I ll make a last check today and will merge today if everything is fine !